docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_vision_feature_label_correlation:
+
+Property Label Correlation
+**************************
+
+This notebook provides an overview for using and understanding the "Property Label Correlation" check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+=================================
+The check estimates for every image property (such as brightness, contrast etc.)
+its ability to predict the label by itself.
+
+This check can help find a potential bias in the dataset, that might lead to shortcut learning - the labels being
+strongly correlated with simple image properties such as color, brightness, aspect ratio and more. This is a critical
+problem, because the model is likely to learn this property instead of the actual visual characteristics of each class,
+as it's easier to do so. In this case, the model will show high performance on images taken in similar conditions, but
+will fail in the wild, where the simple properties don't hold true.
+This kind of correlation will likely stay hidden without this check until tested in the wild.
+
+A famous example is the case of wolves vs. dogs classification, where a model needs to classify whether an image
+contains a wolf or a dog, and can learn to do it by the background instead of the actual animal - in the dataset
+most of the wolves were photographed in the snow and therefore had a white background while all the dogs were
+photographed in the grass and therefore had a green background.
```

Comment:
Cool. Could be nice to add an external link to "more details about this case" (just for the curious reader...) and / or embed an image that demonstrates this. If you find something - I think this can really improve this page (make it look nicer and compelling, and also help in understanding better - since this is a very heavy page with lots of text)

Reply:
@shir22 I tried to add a code that plots some images but it didn't actually plot them in the HTML so I removed the code.

Reply:
why not add a static image in the doc? (why is a code that plots them preferable?)

Reply:
Personally I couldn't find an image with a clear license. Let's consider adding one that we generate from the dataset.

Reply:
Will be done in https://github.com/deepchecks/deepchecks/issues/1929